visualization/app/app.ts

```diff
@@ -4,9 +4,10 @@ import "angular-aria"
 import "angular-material"
 import "./codeCharta/codeCharta.module"
 import "./assets/icon.ico"
+import "angular-sanitize"
 import "./app.scss"
 
-angular.module("app", ["app.codeCharta", "ngMaterial"])
+angular.module("app", ["app.codeCharta", "ngMaterial", "ngSanitize"])
```

Comment:
does the `ngSanitize` do anything? I think it allows to use the `$sanitize` Service. If this isn't used yet, we should remove the dependency `angular-sanitize`

Reply:
It is needed to use htmlContext for md-dialog
Removing it will result in an error